src/align.rs

```diff
@@ -87,3 +90,79 @@ def_align!(8, Align8);
 def_align!(16, Align16);
 def_align!(32, Align32);
 def_align!(64, Align64);
+
+/// A `Vec` that uses a 64 byte aligned allocation.
+///
+/// Only works with `Copy` types so that we don't have to handle drop logic.
```

Comment:
As discussed in the sync meeting, we're going to hold off on `Drop` support since we don't seem to need it for any of the types we expect to put in an `AlignedVec64`. We can revisit this if we end up needing drop support.